README.md

```diff
@@ -1,9 +1,937 @@
 # Acuity Calendar
 
-### _This package is not production ready. It will be changing extensively. Use at your own risk._
+_This package is not production ready. It will be changing extensively. Use at your own risk._
 
 Acuity calendar is an open source set of react calendar components used in the [Acuity Scheduling](https://acuityscheduling.com) app.
 
-## Contributing
+---
+
+## Table of contents
+
+- [Acuity Calendar](#acuity-calendar)
+  - [Table of contents](#table-of-contents)
+  - [Installation](#installation)
+  - [Anatomy](#anatomy)
+    - [DayGrid views (Monthly)](#daygrid-views-monthly)
+    - [TimeGrid views (Daily/Weekly)](#timegrid-views-dailyweekly)
+  - [Datatypes](#datatypes)
+    - [Date](#date)
+    - [Event](#event)
+    - [View](#view)
+    - [Step details](#step-details)
+  - [Usage](#usage)
+    - [Components](#components)
+      - [CalendarGroups](#calendargroups)
+      - [CalendarMonth](#calendarmonth)
+      - [CalendarMonthHeatmap](#calendarmonthheatmap)
+      - [CalendarWeek](#calendarweek)
+      - [Column](#column)
+      - [FullCalendar](#fullcalendar)
+      - [DayList](#daylist)
+      - [DayGrid](#daygrid)
+      - [TimeGrid](#timegrid)
+      - [Toolbar](#toolbar)
+  - [Storybook](#storybook)
+  - [Testing](#testing)
+
+---
+
+## Installation
+
+Currently, this package is only hosted on Github and can be installed by invoking
+
+```bash
+npm i github:AcuityScheduling/acuity-calendar#<commit sha>
+```
+
+In the future, we will be migrating to a semvar system and using git tagging to reference releases.
```

Comment:
as discussed with @bjenkins24

Reply:
nice, I was going to ask about versioning 👍